src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -987,8 +1058,14 @@ void FieldProps::scanREGIONSSection(const REGIONSSection& regions_section) {
 
     for (const auto& keyword : regions_section) {
         const std::string& name = keyword.name();
-        if (keywords::REGIONS::int_keywords.count(name) == 1 || keywords::isFipxxx(name)) {
-            this->handle_int_keyword(keyword, box);
+        if (keywords::REGIONS::int_keywords.count(name)) {
+            this->handle_int_keyword(keywords::REGIONS::int_keywords.at(name), keyword, box);
+            continue;
+        }
+
+        if (keywords::isFipxxx(name)) {
+            auto kw_info = keywords::keyword_info<int>{.scalar_init = 1};
```

Comment:
You can't use designated initializers.

Reply:
Thank's for pointing out - it has popped up again through a rebase probably; fixed now.